Tests/ArcGISToolkitTests/PreplannedMapModelTests.swift

```diff
@@ -17,6 +17,8 @@ import ArcGIS
 @testable import ArcGISToolkit
 
 private extension PreplannedMapAreaProtocol {
+    var mapArea: ArcGIS.PreplannedMapArea? { nil }
```

Comment:
```suggestion
    var mapArea: PreplannedMapArea? { nil }
```
Also, the ArcGIS prefix for `LoadableImage` can be removed as well.